go.mod

```diff
@@ -3,27 +3,70 @@ module github.com/rancher/k3d
 go 1.12
 
 require (
-	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
-	github.com/Microsoft/go-winio v0.4.12 // indirect
-	github.com/containerd/containerd v1.2.7 // indirect
-	github.com/docker/distribution v2.7.1+incompatible // indirect
-	github.com/docker/docker v0.7.3-0.20190723064612-a9dc697fd2a5
+	github.com/docker/docker v0.7.3-0.20190731001754-589f1dad8dad
 	github.com/docker/go-connections v0.4.0
-	github.com/docker/go-units v0.3.3 // indirect
-	github.com/gogo/protobuf v1.2.1 // indirect
-	github.com/gorilla/mux v1.7.3 // indirect
 	github.com/mattn/go-runewidth v0.0.4 // indirect
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/morikuni/aec v0.0.0-20170113033406-39771216ff4c // indirect
 	github.com/olekukonko/tablewriter v0.0.1
-	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
-	github.com/opencontainers/image-spec v1.0.1 // indirect
-	github.com/pkg/errors v0.8.1 // indirect
+	github.com/rancher/k3s v1.0.1
```

Comment:
This seems to pull in lots of dependencies O.o

Reply:
Yeah, this is a dependency hell... I tyhink it would be better to just copy&paste the contents of [registry.go](https://github.com/rancher/k3s/blob/master/pkg/agent/templates/registry.go) and keep the dependencies as they are. What do you think?

Reply:
I guess that makes sense in this trade-off :thinking: 
But I also didn't look for a different solution yet.